project/client/js/InventoryHelper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Created by vova on 06.10.2015.
+ */
+Template.inventory.helpers({
+    problem: function(){
```

Comment:
useless and mismatching the return value. You name this as problem, but return problems;